domain-ee/ee-ep-merge-app/src/python_src/model/contention.py

```diff
@@ -21,7 +21,7 @@ class ContentionSummary(BaseModel):
     notification_date: datetime | None = None
     contention_type_code: str
     classification_type: int | None = None
-    diagnostic_type_code: str | None = None
+    diagnostic_type_code: int | None = None
```

Comment:
Assuming this field is for the diagnostic code -- is there a guarantee it'll be an int now and in the future? Asking because sometimes diagnostic codes aren't ints because they're hyphenated.